wordpress-mobile/WordPress-Android
# Support: JP migration FAQ card

This PR introduces migration FAQ card on JP Help screen

Fixes #18025 

|FAQ card|FAQ page|
|-|-|
|![Screenshot_20230303_133955](https://user-images.githubusercontent.com/990349/222618857-0ce7d3a8-000f-4454-86ee-39d04dbcd863.png)|![Screenshot_20230302_153546](https://user-images.githubusercontent.com/990349/222332237-33d625e3-080d-41f5-9556-e3f5aabec933.png)|


To test:

Test 1
- Delete any JP app instance that you might have
- Launch WP app and log-in
- Tap on Switch to the new Jetpack app button or overlay from WP app to install JP app
- Install JP app and complete migration ( Alternatively, try installing JP app using AS)
- Navigate to Me > Help on JP app
- Verify the FAQ card is shown on the help screen
- Tap on the card/row, ensure it shows Switch to the Jetpack app page

Test 2
- Navigate to Me > Help on WP app
- Ensure the FAQ card is not shown

Test 3
- Look for the following tracking in the Logcat of AS.  Alternatively, view this through Me > Help > Logs
🔵 Tracked: support_migration_faq_viewed
🔵 Tracked: support_migration_faq_tapped

> **Note**
> FAQ only appears when migration is completed.  Layout on Android varies from that of iOS to match for the platform

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit test

PR submission checklist:
